src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body,
+ul {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  text-transform: uppercase;
+  line-height: 14px;
+  font-size: 12px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__link {
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  height: 60px;
+  display: flex;
+  margin-right: 20px;
+}
+
+.logo {
+  display: inline-flex;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__item:last-of-type .nav__link {
```

Comment:
You may try to find a better solution here.
For example: `.nav__link .nav__link--last`